Doc 896 enterprise license auto renew env variable description update (#2143)
n8n-io/n8n-docs: docs/hosting/configuration/environment-variables/licenses.md

| :------- | :---- | :------- | :---------- |
| `N8N_HIDE_USAGE_PAGE` | boolean | `false` | Hide the usage and plans page in the app. |
| `N8N_LICENSE_ACTIVATION_KEY` | String | `''` | Activation key to initialize license. Not applicable if the n8n instance was already activated. |

> | `N8N_LICENSE_AUTO_RENEW_ENABLED` | Boolean | `true` | Enables (true) or disables (false) autorenewal for licenses.<br>Failure to renew the license will disable Enterprise features.<br>If disabled, the license will need to be manually renewed every 10 days by navigating to Usage and Plan page and pressing F5. |

Reviewer: ```suggestion
| `N8N_LICENSE_AUTO_RENEW_ENABLED` | Boolean | `true` | Enables (true) or disables (false) autorenewal for licenses. If disabled, you need to be manually renew the license every 10 days by navigating to **Settings** > **Usage and plan**, and pressing `F5`.<br>Failure to renew the license will disable Enterprise features.<br> |
```

> | `N8N_LICENSE_AUTO_RENEW_ENABLED` | Boolean | `true` | Enables (true) or disables (false) autorenewal for licenses. If disabled, you need to manually renew the license every 10 days by navigating to **Settings** > **Usage and plan**, and pressing `F5`.<br>Failure to renew the license will disable Enterprise features. |

Reviewer: ```suggestion
| `N8N_LICENSE_AUTO_RENEW_ENABLED` | Boolean | `true` | Enables (true) or disables (false) autorenewal for licenses. <br>If disabled, you need to manually renew the license every 10 days by navigating to **Settings** > **Usage and plan**, and pressing `F5`. Failure to renew the license will disable Enterprise features. |
```

> | `N8N_LICENSE_AUTO_RENEW_ENABLED` | Boolean | `true` | Enables (true) or disables (false) autorenewal for licenses. If disabled, you need to be manually renew the license every 10 days by navigating to **Settings** > **Usage and plan**, and pressing `F5`.<br>Failure to renew the license will disable Enterprise features. |

Reviewer: ```suggestion
| `N8N_LICENSE_AUTO_RENEW_ENABLED` | Boolean | `true` | Enables (true) or disables (false) autorenewal for licenses. If disabled, you need to manually renew the license every 10 days by navigating to **Settings** > **Usage and plan**, and pressing `F5`.<br>Failure to renew the license will disable Enterprise features. |
```
